spec/models/manageiq/providers/ibm_power_vc/cloud_manager/refresher_spec.rb

```diff
@@ -21,66 +38,67 @@ def assert_ems
       expect(ems.last_refresh_error).to be_nil
       expect(ems.last_refresh_date).not_to be_nil
 
-      expect(ems.availability_zones.count).to eq(3)
-      expect(ems.cloud_tenants.count).to eq(1)
-      expect(ems.flavors.count).to eq(12)
-      expect(ems.vms.count).to eq(4)
+      expect(ems.cloud_tenants.count).to be > 1
+      expect(ems.availability_zones.count).to be > 2
+      expect(ems.flavors.count).to be > 2
+      expect(ems.vms.count).to be > 1
```

Comment:
If we're creating these resources should we know how many there are?  I get flavors come built-in and so can change without changes to the terraform spec but vms we should know the number of right?

Reply:
And if you want to keep it more dynamic if we do https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/pull/31/files#r766843742 you could do `expect(ems.flavors.count).to eq(resources["openstack_compute_flavor_v2"].count)`

Reply:
The problem with this was that [scnlpowercloud](https://scnlpowercloud.pok.stglabs.ibm.com/) is heavily used. As a result the number of flavors, images, VMs kept on changing.
I am trying to get our own project  **miq-test-env** in [scnlpowercloud](https://scnlpowercloud.pok.stglabs.ibm.com/)  so we can have definite numbers and can control it.